hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/scope/HandleHederaOperationsTest.java

```diff
@@ -417,7 +417,9 @@ void createContractWithSelfAdminParentDispatchesAsExpectedThenMarksCreated() thr
         final var pendingId = ContractID.newBuilder().contractNum(666L).build();
         final var synthContractCreation = synthContractCreationFromParent(pendingId, parent)
                 .copyBuilder()
-                .adminKey((Key) null)
+                .adminKey(Key.newBuilder()
```

Comment:
Sad that this is the only unit test affected by this change, and that only incidently.

**RANT ON**

The base class `HandleHederaOperations` is basically untestable via unit tests - notwithstanding that this test file has 30 separate tests. Let me give you an example - here's a call chain:

* `public createContract` -> **`private`** `dispatchAndMarkCreation` -> **`private`** `contractBodyCustomizerFor` -> **`private`** `standardized` -> **`private`** `needsStandardization`

10 separate conditional choices are in that path (`if` statements, two fluent `fooOrElse()` methods, a trinary op, and a boolean expression with 3 parts separated by `||`).

Overuse of `private` probably indicates some design-in-the-local problem in this class, huh?

**RANT OFF**

(This isn't a _Hedera_ problem.  It's an industry problem.  I mean: I've seen it elsewhere.  A lot.)